pontoon/base/signals.py

```diff
@@ -54,53 +54,73 @@ def project_deleted(sender, **kwargs):
             locale.aggregate_stats()
 
 
+def create_group(instance, group_name, perms, name_prefix):
+    """
+    Create all objects related to a group of users, e.g. translators, managers.
+    """
+    ct = ContentType.objects.get(app_label='base', model=instance.__class__.__name__.lower())
+    group, _ = Group.objects.get_or_create(name='{} {}'.format(name_prefix, group_name))
+
+    for perm_name in perms:
+        perm = Permission.objects.get(content_type=ct, codename=perm_name)
+        group.permissions.add(perm)
+
+        setattr(instance, '{}_group'.format(group_name), group)
+
+
+def assign_group_permissions(instance, group_name, perms):
```

Comment:
Can't we join this function with `create_group`? And then we could also get rid of `assign_locale_group_permissions` and `assign_project_locale_group_permissions`.